spec/system/performance_tracking/viewing_spec.rb

```diff
@@ -42,5 +42,20 @@
         expect(heading_text).to eq('Performance tracking')
       end
     end
+
+    context 'when user managers are logged in on staging' do
+      before do
+        # Override allow_user_managers_service_access as per staging
```

Comment:
No need to comment here imo, Line 49 is fairly self-explanatory imo. Use comments to justify code decisions/explain _why_ something works in a particular way, let the code explain _how_ that was done

Reply:
Agree with this.